coderd/oauth2_metadata_test.go

```diff
@@ -41,3 +41,35 @@ func TestOAuth2AuthorizationServerMetadata(t *testing.T) {
 	require.Contains(t, metadata.GrantTypesSupported, "refresh_token")
 	require.Contains(t, metadata.CodeChallengeMethodsSupported, "S256")
 }
+
+func TestOAuth2ProtectedResourceMetadata(t *testing.T) {
+	t.Parallel()
+
+	client := coderdtest.New(t, nil)
+
+	ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
+	defer cancel()
+
+	// Get the protected resource metadata
+	resp, err := client.Request(ctx, http.MethodGet, "/.well-known/oauth-protected-resource", nil)
```

Comment:
It's not immediately obvious whether this client is authenticated or not. If it's not meant to require authentication, I'd suggest testing with a new HTTP client.

Reply:
done.